locksmith/src/app.ts

```diff
@@ -7,9 +7,12 @@ import * as Tracing from '@sentry/tracing'
 import cookieParser from 'cookie-parser'
 import router from './routes'
 import { errorHandler } from './utils/middlewares/error'
+import timeout from 'connect-timeout'
 
 const app = express()
 
+app.use(timeout('3s'))
```

Comment:
Add this to config file and increase it to 25s? We have some slow endpoints. Until we can optimize them, shouldn't break them.

Reply:
ok I think we should try to identify these slow endpoints and see how to speed them up :)